package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -222,6 +238,9 @@ define([
               paramDefn.autoSubmitUI = autoSubmit;
             }
 
+            if (this._oldParameterDefinition == null) {
+              this.extractDefaultValues(paramDefn);
+            }
```

Comment:
@tmorgner 
My bad, this scenario slipped under my radar. Pull-requests updated.